experimental/traffic-portal/src/app/core/cache-groups/asns/detail/asn-detail.component.scss

```diff
@@ -12,15 +12,17 @@
 * limitations under the License.
 */
 
-mat-card {
+.mat-mdc-card {
```

Comment:
according to [the Angular Material docs on theming](https://material.angular.io/guide/theming#style-customization-outside-the-theming-system):

> Component DOM structure and CSS classes are considered private implementation details that may change at any time.

The examples for this component do not include this class selector; it seems totally undocumented. They also don't use `mat-card`, though. Instead they always apply a class to the card and use the selector for that class. I'm not sure how this internal refactor was applied throughout _our_ code, but I don't think this change should be made.

Reply:
I would agree if the automatic part of the migration didn't make changes to classes that would also be considered "internal" instead of leaving a comment. For instance `.mat-button-base` -> `.mat-mdc-button-base` and `.mat-fab` -> `.mat-mdc-fab` from [here in this commit](https://github.com/apache/trafficcontrol/pull/7454/commits/366d8a808397e1c76cd7e97c738177c632da7c55#diff-6b6bcdd753b1d408597417c4e69e3a84dde884ccff7b382eb3b9602d9cabacb8)

Reply:
I suppose then that they could all be fixed at one time together. Personally I think it'd be easier to limit the impact to only those that are already using internal classes right now, but ultimately that doesn't really matter. Would you mind opening an issue so we don't forget about this?

Reply:
Will do